### PR TITLE
Update admin-api.md

### PR DESCRIPTION
**Pull request by ridvanaltun**
- fix typo
- correct some infos

**PS:**  I do not pay attention to the names of the commit I throw. Please merge by squashing.

### CODE REVIEW

**Comment by hishamhm** [comment]
@ridvanaltun Thank you for the PR! We are going to apply these changes, but since the changes are in a generated file, we need to port these fixes over to the appropriate files in the Kong/kong repository: https://github.com/Kong/kong/blob/master/scripts/autodoc-admin-api and https://github.com/Kong/kong/blob/master/autodoc/data/admin-api.lua

@GayleNeumann you can hand this one over to us, we'll take it from here! We will close it once the changes are merged on our end, so please don't merge this one.

File: app/2.0.x/admin-api.md [hunk 1]
```diff
@@ -972,8 +972,8 @@ HTTP 200 OK
         },
         ...
       ],
-      "offset" = "1fb491c4-f4a7-4bca-aeba-7f3bcee4d2f9",
-      "next" = "/tags/example?offset=1fb491c4-f4a7-4bca-aeba-7f3bcee4d2f9",
+      "offset": "1fb491c4-f4a7-4bca-aeba-7f3bcee4d2f9",
+      "next": "/tags/example?offset=1fb491c4-f4a7-4bca-aeba-7f3bcee4d2f9"
```
Comment (kikito) [inline app/2.0.x/admin-api.md:976]: These changes are now included in https://github.com/Kong/kong/pull/5870

File: app/2.0.x/admin-api.md [hunk 2]
```diff
@@ -1964,32 +1964,32 @@ HTTP 201 Created
 
 ##### List Plugins Associated to a Specific Route
 
-<div class="endpoint get indent">/routes/{route id}/plugins</div>
+<div class="endpoint get indent">/routes/{route name or id}/plugins</div>
```
Comment (kikito) [inline app/2.0.x/admin-api.md:1967]: I have tried adding these changes to the templates we use to generate the docs but it is harder than anticipated. Leaving this PR open as a reminder that this needs to happen.

**Comment by kikito** [comment]
The second part of this PR is now done in automation via #2113

**Pull request closed without merging**